spec/foundation_rails_helper/flash_helper_spec.rb

```diff
@@ -10,6 +10,8 @@
   include ActionView::Helpers::FormTagHelper
   include FoundationRailsHelper::FlashHelper
 
+  RSpec::Matchers.define_negated_matcher :not_have_css, :have_css
```

Comment:
Is this necessary? Could we use the existing negated matcher `have_no_css` or even `to_not have_css`?